frontend/simple/views/components/Voting/Sign.vue

```diff
@@ -103,16 +106,16 @@ export default {
       }
     },
     isTypeMember () {
-      return this.type === 'member'
+      return ['invitation', 'removal'].includes(this.type)
     },
     isTypeRule () {
-      return this.type === 'rule'
+      return ['changeThreshold', 'memberApprovalThreshold', 'memberRemovalThreshold'].includes(this.type)
     },
     isTypeMincome () {
-      return this.type === 'mincome'
+      return this.type === 'incomeProvided'
     },
     isTypeRuleOrMincome () {
-      return ['rule', 'mincome'].includes(this.type)
+      return ['changeThreshold', 'memberApprovalThreshold', 'memberRemovalThreshold', 'incomeProvided'].includes(this.type)
```

Comment:
Took me a while to understand where these values came from. Could we have this types in a file and reuse them where needed? Similar to `countdownStates.js`?

Similar to what's done on the validators votingType.

-- EDIT --

Similar to this: https://github.com/okTurtles/group-income-simple/pull/425/files#diff-3103521226e9a65bc929902e992ef919R196

or 

https://github.com/okTurtles/group-income-simple/pull/425/files#diff-3103521226e9a65bc929902e992ef919R240

Reply:
Yes, types definitely need to be defined and grouped together as constants for this reason.

Also, `isTypeRuleOrMincome` is a violation of DRY, better to simply do `isTypeRule() || isTypeMincome()` in the markup, or if for some reason that's not OK do it as that here in the JS.